pkg/util/portforward/portforward.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package portforward
```

Comment:
FYI: I overwrote this with the one from loftctl. I guess this is the more recent version, as it's using ctx etc?!